ui/helpers/url.js

```diff
@@ -120,3 +123,5 @@ export const bugzillaBugsApi = function bugzillaBugsApi(api, params) {
 };
 
 export const deployedRevisionUrl = '/revision.txt';
+
+export const loginCallbackUrl = '/login.html';
```

Comment:
`'/login.html'` is used in a couple more places - could you update those too?
* `treeherder/ui/js/components/auth.js`
* `treeherder/ui/js/auth/auth-utils.js`